theme/toc.html

```diff
@@ -1,24 +1,41 @@
+{# loop through all navigation items #}
+{# if the current item has children #}
 {% if nav_item.children %}
+  {# if the current item title does not start with _ #}
```

Comment:
Are those comments, the lines that start with `#`? Are they supposed to be in this PR?

Reply:
Yes they are, and yes - I had a really hard time to understand the code in the beginning. Documented it for myself, and thought it might be nice to have that in general. 

Should I strip them?